RubberduckTests/Inspections/AssignedByValParameterInspectionTests.cs

```diff
@@ -97,19 +97,152 @@ public void AssignedByValParameter_ReturnsResult_SomeAssignedByValParams()
         [TestCategory("Inspections")]
         public void AssignedByValParameter_QuickFixWorks()
```

Comment:
I'd say were OK without it.  *Technically* bracketed identifiers (foreign identifiers) are not specifically disallowed by the language specification:

```
3.3.5.2 Reserved Identifiers and IDENTIFIER

    reserved-identifier = Statement-keyword / marker-keyword / operator-identifier / 
            special-form / reserved-name / literal-identifier / rem-keyword / 
            reserved-for-implementation-use / future-reserved 

    IDENTIFIER = <any lex-identifier that is not a reserved-identifier>
```
That would make it implementation specific, _but_... I'm not aware of any other implementations for it to be specific to, and none of the Office hosts allow it.  Unsure about VB6, but this will be the least of our worries for VB6 compatibility...